addons/web/static/tests/views/calendar/helpers.js

```diff
@@ -501,6 +501,37 @@ export async function resizeEventToTime(target, eventId, dateTime) {
     await nextTick();
 }
 
+export async function resizeEventToDate(target, eventId, date) {
+    const event = findEvent(target, eventId);
+    const slot = findAllDaySlot(target, date);
+
+    await scrollTo(event);
+    await triggerEventForCalendar(event, "mouseenter");
+
+    // Find event resizer
+    const resizer = event.querySelector(".fc-end-resizer");
+    resizer.style.display = "block";
+    resizer.style.width = "100%";
+    resizer.style.height = "1em";
+    resizer.style.bottom = "0";
```

Comment:
I am not sure to understand why we need to add those custom stylings.

Reply:
Hello @xavierbol 

We're not giving styling to `pill` but we're giving styling to `.fc-end-resizer`
which won't be visible in the UI until we hover at the end of the pill.
now the thing is by giving height and width we're making sure that
`.fc-end-resizer` is at the end of a pill which will be `resizerPos`.

(we can do this even by reducing/adding a few degrees from "X" and "Y" of `resizerPos` but wouldn't it
be inconvenient to do? :thinking: )

removing this styling will lead to an Error where "mousedown won't be able to find the correct 
target".
let me know if there is a better solution to this that I'm not aware of. :thinking: 
Thanks :slightly_smiling_face: